docs/recipes/install/rocky9/x86_64/warewulf4/slurm/steps.tex

```diff
@@ -0,0 +1,298 @@
+\input{common/legal}
+
+\newpage
+\tableofcontents
+\newpage
+
+% Introduction  --------------------------------------------------
+
+\section{Introduction} \label{sec:introduction}
+\input{common/install_header}
+\input{common/intro} \\
+
+\input{common/base_edition/edition}
+\input{common/audience}
+\input{common/requirements}
+\input{common/inputs}
+
+% begin_ohpc_run
+% ohpc_validation_newline
+% ohpc_validation_comment Verify OpenHPC repository has been enabled before proceeding
+% ohpc_validation_newline
+% ohpc_command dnf repolist | grep -q OpenHPC
+% ohpc_command if [ $? -ne 0 ];then
+% ohpc_command    echo "Error: OpenHPC repository must be enabled locally"
+% ohpc_command    exit 1
+% ohpc_command fi
+% end_ohpc_run
+
+% Base Operating System --------------------------------------------
+
+\section{Install Base Operating System (BOS)}
+\input{common/bos}
+
+%\clearpage
+% begin_ohpc_run
+% ohpc_validation_newline
+% ohpc_validation_comment Disable firewall
+\begin{lstlisting}[language=bash,keywords={}]
+[sms](*\#*) systemctl disable --now firewalld
+\end{lstlisting}
+% end_ohpc_run
+
+% ------------------------------------------------------------------
+
+\section{Install \OHPC{} Components} \label{sec:basic_install}
+\input{common/install_ohpc_components_intro.tex}
+
+\subsection{Enable \OHPC{} repository for local use} \label{sec:enable_repo}
+\input{common/enable_ohpc_repo}
+\input{common/rocky_repos}
+\input{common/automation}
+
+
+\subsection{Add provisioning services on {\em master} node} \label{sec:add_provisioning}
+\input{common/install_provisioning_warewulf4_intro}
+\input{common/enable_pxe}
+\input{common/time}
+
+\vspace*{0.15cm}
+\subsection{Add resource management services on {\em master} node} \label{sec:add_rm}
+\input{common/install_slurm}
+
+\subsection{Optionally add \InfiniBand{} support services on {\em master} node} \label{sec:add_ofed}
+\input{common/ibsupport_sms_centos}
+
+\subsection{Optionally add \OmniPath{} support services on {\em master} node} \label{sec:add_opa}
+\input{common/opasupport_sms_centos}
+
+\vspace*{-0.15cm}
+\subsection{Complete basic Warewulf setup for {\em master} node} \label{sec:setup_ww}
+\input{common/warewulf4_setup}
+\input{common/warewulf4_setup_centos}
+
+\subsection{Define {\em compute} image for provisioning}
+\input{common/warewulf4_mkchroot_rocky}
+
+\subsubsection{Add \OHPC{} components} \label{sec:add_components}
+\input{common/warewulf4_add_to_compute_chroot_intro}
+
+%\newpage
+% begin_ohpc_run
+% ohpc_validation_comment Add SLURM and other components to compute instance
+\begin{lstlisting}[language=bash,literate={-}{-}1,keywords={},upquote=true,literate={BOSVER}{\baseos{}}1]
+[sms](*\#*) wwctl container exec rocky-9.4 /bin/bash <<- EOF 
+  # Add Slurm client support meta-package and enable munge and slurmd
+  dnf -y install ohpc-slurm-client
+  systemctl enable munge
+  systemctl enable slurmd
+
+  # Add Network Time Protocol (NTP) support
+  dnf -y intall chrony
```

Comment:
install

Reply:
Obviously my local testing is less than optimal, and my spelling 😁